### PR TITLE
Screenshot Create New Mock added to combine steps 4-7

### PR DESCRIPTION
**Pull request by maksvet**
- screenshot for Create New Mock page added
- Title, openApi petstore.json and create button highlighted on screenshot
- steps rearranged

### CODE REVIEW

**Comment by ReubenKogie** [comment]
Hey @maksvet, any pics please to attach to the PR to show how it looks in the webpage?

**Comment by maksvet** [comment]
> 

Good point.
image

File: Gemfile [hunk 1]
```diff
@@ -15,3 +15,5 @@ group :jekyll_plugins do
   gem "jekyll-include-cache"
   gem "jekyll-algolia"
 end
+
+gem "webrick", "~> 1.7"
```
Comment (ReubenKogie) [inline Gemfile:19]: Might be worth placing this change in a separate commit as it's a missing package/dependency? - Already set one up on branch `hotfix/update-webrick-dependency` via PR
